src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -84,7 +84,7 @@ export const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
       break;
 
     case VenueTemplate.embeddable:
-      template = <Embeddable venue={venue} />;
+      template = <Embeddable space={venue as SpaceWithId} />;
```

Comment:
Let's replace the `venue: WithId<AnyVenue>;`  with `SpaceWithId` in the `TemplateWrapperProps` a little above in this file
it would effectively remove the necessity of using these `as`

Reply:
done [d177469](https://github.com/sparkletown/sparkle/pull/2996/commits/d177469c3d53acd88f2d7f46a96937a2b41bd8f3)